app/views/activate_trial.erb

```diff
@@ -11,3 +11,8 @@
     <a href="/"> No Thanks </a>
   </div>
 </div>
+<script type="text/javascript">
```

Comment:
I figured when we deprecate the erb templates then we will use an Ember action to handle it.